### ISSUE TITLE
video request

### ISSUE THREAD

**Issue by HumHongeKamyaab** [daviwil/runemacs#4]
I dont know if this is right forum for video requests, but can you make a similar video on `exwm` and its integration with linux distribution.

**Comment by daviwil** [comment]
It's a great place for video requests!  I am definitely planning to do at least one video on EXWM and how to set up an entire desktop environment with it as I've done in my own configuration.  It'll probably take a month or two before we get to it, but I'm excited to cover it!

**Comment by HumHongeKamyaab** [comment]
@daviwil I tried to replicate yr `exwm` config with `polybar` in my Ubuntu based OS with Gnome DE. Its taking some effort, as I don't know anything about Guix OS. May be it will better to make a video integrating common GNU/linux OS like Ubuntu.

**Comment by daviwil** [comment]
Yep, that's the plan, use a common Linux distribution for the example and then later talk about how to use Guix with everything.

**Comment by HumHongeKamyaab** [comment]
I am really fascinated about Guix after reading yr config.  Will try to learn more about it, and replicate yr config.
May be it sounds greedy, but a video about how you are using Guix OS with your config file will be really helpful.

**Comment by daviwil** [comment]
I definitely want to talk about Guix after I've covered a lot of Emacs topics because it basically lets you apply an Emacs-like configuration to your entire system.
